content/docs/command-reference/remote/modify.md

```diff
@@ -218,8 +218,8 @@ these settings, you could use the following options:
 To communicate with a remote object storage that supports an S3 compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.) you
-must explicitly set the `endpointurl` in the configuration:
+[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.)
+please set the `endpointurl` explicitly in the configuration:
```

Comment:
`please` - we don't usually use that kind of language, especially in the command reference

Reply:
OK, updated. But there's 23 more instances of the word in the cmd ref, actually 🙂